tests/30rooms/06invite.pl

```diff
@@ -106,3 +113,36 @@
          Future->done(1);
       });
    };
+
+my $other_local_user_preparer = local_user_preparer();
+
+test "Invited user can reject invite",
+   requires => [ do { my $creator = local_user_preparer(); $creator, local_user_preparer(), inviteonly_room_preparer( creator => $creator ) } ],
```

Comment:
Some more linefeeds would help here. Maybe

    requires => [ local_user_preparer(),
       do {
          my $creator = local_user_preparer();
          $creator, inviteonly_room_preparer( creator => $creator );
       } ],

by pulling the (unrelated) invitee up front. You'll have to swap the order of `$creator` and `$invitee` to the invoked function below then.